assets/src/stories-editor/components/inserter/index.js

```diff
@@ -127,12 +126,8 @@ Inserter.propTypes = {
 const applyWithSelect = withSelect( ( select ) => {
 	const { isReordering } = select( 'amp/story' );
```

Comment:
Why is this still down here and not inside the component?

Reply:
For use by the `ifCondition` HOC. Felt a bit easier with that, but can also use hooks instead.